### PR TITLE
Feature/fix behondhd

### PR DESCRIPTION
**Pull request by p0psicles**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: medusa/providers/generic_provider.py [hunk 1]
```diff
@@ -829,14 +829,25 @@ def add_cookies_from_ui(self):
...
-        if not all(req_cookie in [x.rsplit('=', 1)[0] for x in self.cookies.split(';')]
-                   for req_cookie in self.required_cookies):
-            return {
-                'result': False,
-                'message': "You haven't configured the requied cookies. Please login at {provider_url}, "
-                           'and make sure you have copied the following cookies: {required_cookies!r}'
-                           .format(provider_url=self.name, required_cookies=self.required_cookies)
-            }
+        if self.required_cookies:
+            if self.name != 'Beyond-HD':
```
Comment (p0psicles) [inline medusa/providers/generic_provider.py]: For now beyond-hd is the only one with a cookie name like: remember_web_59ba36addc2b2f9401580f01445acd8ea4e30af9d. So I had to make an exception for this provider.

File: medusa/providers/torrent/html/beyondhd.py [hunk 2]
```diff
@@ -111,14 +112,17 @@ def parse(self, data, mode):
                 cells = result('td')
 
                 try:
-                    link = cells[1].find('div')
+                    if len(cells) < 2:
```
Comment (p0psicles) [inline medusa/providers/torrent/html/beyondhd.py]: There is an hidden tr at the end with one td.

**Comment by p0psicles** [comment]
don't merge

**Comment by emaw003** [comment]
Hey. I'm getting the error: [80b323e] Cookie is not correctly formatted: **********.

I'm just copying and pasting the "remember_web_xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx" from BHD. Am I doing something wrong? Thanks!

**Comment by p0psicles** [comment]
It's key=value. You probably only have the key.
But. It's already reported the cookie is for some reason expiring after one day. We still need to look into this.

**Pull request merged**